src/App.tsx

```diff
@@ -14,11 +15,67 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGoods: string,
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    selectedGoods: 'Jam',
```

Comment:
selectedGood - выбранный товар, не выбранные товары
```suggestion
    selectedGood: 'Jam',
```